core/iwasm/compilation/aot_emit_table.c

```diff
@@ -428,7 +485,15 @@ aot_compile_op_table_grow(AOTCompContext *comp_ctx, AOTFuncContext *func_ctx,
     param_types[0] = INT8_PTR_TYPE;
     param_types[1] = I32_TYPE;
     param_types[2] = I32_TYPE;
-    param_types[3] = I32_TYPE;
+#if WASM_ENABLE_GC != 0
+    if (comp_ctx->enable_gc) {
+        param_types[3] = OBJECT_REF_TYPE;
```

Comment:
No need to change for table grow?

Reply:
I just tried to make the function parameter's type match with the `aot_table_grow`. When GC is enabled, the table_elem_type_t is void* rather than unsigned long int. 

When on a 64-bit platform, is it possible to become a problem if don't change it?